HW_1/task2.py

```diff
@@ -0,0 +1,7 @@
+n = int(input('Введите трехзначное целое положительное число: '))
+summ = 0
+while n>0:
+    digit = n%10
+    summ += digit
+    n//=10
```

Comment:
Не забывайте про белый между математическими операторами.
```suggestion
while n > 0:
    digit = n % 10
    summ += digit
    n //= 10
```